benches/bin/bench-tikv.rs

```diff
@@ -43,6 +43,9 @@ mod server;
 mod pd;
 #[path="../../tests/raftstore/pd_ask.rs"]
 mod pd_ask;
+#[allow(unused)]
+#[path="../../tests/raftstore/transport_simulate.rs"]
```

Comment:
if those three line removed, make will complain. I don't fully understand the error log, seems transport_simulate not compiled into the crate. here is the log:

    benches/bin/../../tests/raftstore/server.rs:34:33: 34:51 error: unresolved import `super::transport_simulate::Strategy`. Maybe a missing `extern crate transport_simulate`? [E0432]
    benches/bin/../../tests/raftstore/server.rs:34 use super::transport_simulate::{Strategy, SimulateTransport};
                                                                                   ^~~~~~~~~~~~~~~~~~
    benches/bin/../../tests/raftstore/server.rs:34:33: 34:51 help: run `rustc --explain E0432` to see a detailed explanation
    benches/bin/../../tests/raftstore/server.rs:34:43: 34:61 error: unresolved import `super::transport_simulate::SimulateTransport`. Maybe a missing `extern crate transport_simulate`? [E0432]
    benches/bin/../../tests/raftstore/server.rs:34 use super::transport_simulate::{Strategy, SimulateTransport};
                                                                                             ^~~~~~~~~~~~~~~~~~
    benches/bin/../../tests/raftstore/server.rs:34:43: 34:61 help: run `rustc --explain E0432` to see a detailed explanation
    benches/bin/../../tests/raftstore/cluster.rs:34:5: 34:23 error: unresolved import `super::transport_simulate::Strategy`. Maybe a missing `extern crate transport_simulate`? [E0432]
    benches/bin/../../tests/raftstore/cluster.rs:34 use super::transport_simulate::Strategy;
                                                        ^~~~~~~~~~~~~~~~~~
    benches/bin/../../tests/raftstore/cluster.rs:34:5: 34:23 help: run `rustc --explain E0432` to see a detailed explanation
    benches/bin/../../tests/raftstore/node.rs:33:33: 33:51 error: unresolved import `super::transport_simulate::Strategy`. Maybe a missing `extern crate transport_simulate`? [E0432]
    benches/bin/../../tests/raftstore/node.rs:33 use super::transport_simulate::{Strategy, SimulateTransport};
                                                                                 ^~~~~~~~~~~~~~~~~~
    benches/bin/../../tests/raftstore/node.rs:33:33: 33:51 help: run `rustc --explain E0432` to see a detailed explanation
    benches/bin/../../tests/raftstore/node.rs:33:43: 33:61 error: unresolved import `super::transport_simulate::SimulateTransport`. Maybe a missing `extern crate transport_simulate`? [E0432]
    benches/bin/../../tests/raftstore/node.rs:33 use super::transport_simulate::{Strategy, SimulateTransport};
                                                                                           ^~~~~~~~~~~~~~~~~~
    benches/bin/../../tests/raftstore/node.rs:33:43: 33:61 help: run `rustc --explain E0432` to see a detailed explanation
    error: aborting due to 5 previous errors